core/amber/src/main/scala/edu/uci/ics/texera/web/storage/MongoDatabaseManager.scala

```diff
@@ -26,4 +27,21 @@ object MongoDatabaseManager {
     database.listCollectionNames().into(new util.ArrayList[String]()).contains(collectionName)
   }
 
+  def getDatabaseSize(collectionNames: Array[mongoStorage]): Array[mongoStorage] = {
+    var count = 0
+
+    for (collection <- collectionNames) {
+      val stats: Document = database.runCommand(new Document("collStats", collection.pointer))
+      collectionNames(count) = mongoStorage(
+        collection.workflowName,
+        stats.getInteger("totalSize") / 1,
```

Comment:
Is there a reason behind the `/ 1` ? If so, can you add some comments here?

Reply:
It was used to convert integer to double. Now I use the toDouble method to convert integer to double.